docs-parts/queries/03-Fetch_lang1.rst

```diff
@@ -35,6 +35,42 @@ Primary key values
 
 ``KEY`` can also used when returning attribute values as separate variables, such that one of the returned variables contains the entire primary keys.
 
+Sorting and limiting the results
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To sort the result, use the ``order_by`` keyword argument.
+
+.. code-block:: python
+
+    # ascending order:
+    data = query.fetch(order_by='name')
+    # descending order:
+    data = query.fetch(order_by='name desc')  
+    # by name first, year second:
+    data = query.fetch(order_by=('name desc', 'year'))
+    # sort by the primary key:
+    data = query.fetch(order_by='KEY')
+    # sort by name but for same names order by primary key:
+    data = query.fetch(order_by=('name', 'KEY desc'))
+
+The ``order_by`` argument can be a string specifying the attribute to sort by. By default the sort is in ascending order. Use ``'attr desc'`` to sort in descending order by attribute ``attr``.  The value can also be a sequence of strings, in which case, the sort performed on all the attributes jointly in the order specified.
+
+The special attribute name ``"KEY"`` represents the primary key attributes in order that they appear in the index. Otherwise, this name can be used as any other argument.
```

Comment:
```suggestion
The special attribute name ``'KEY'`` represents the primary key attributes in order that they appear in the index. Otherwise, this name can be used as any other argument.
```